.github/workflows/publish.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
   build-publish-docker-images:
     runs-on: ubuntu-latest
-    needs: get-version
+    needs: [get-version, build_wheels]
```

Comment:
should be after the publish sdk step not build wheels

Reply:
Oh right fixed

Reply:
Good catch